### ISSUE TITLE
Returned products on WC Reports

### ISSUE THREAD

**Issue by zokky93** [woothemes/woocommerce#10165]
Hi. 

I noticed an issue on WooCommerce. When I make a sale, and it contains 3 products for example, but the client returns 1 product from 3 and I go to the order setting and put one product as returned from that order, everything looks fine instead of the reports. 

On the reports, the value of the order is still 100%, not updated with the returned product. How can I solve this?

Regards

**Comment by mikejolley** [comment]
If you refund a product during a different date range, the refund won't appear when the original order was made. e.g. If I have a weekly report, had an order last week, and refunded this week, this week would show a negative total, and last week would be unchanged.

Thats the only thing you may come across.

**Comment by zokky93** [comment]
Hi. Thanks for your answer but Ive tried to make a new order, for 2 bracelets for example. I went and refunded one of it because the client didn't wanted it anymore, and then set the order as finalized. I got this http://d.pr/i/1gqtm. All good until here. 

But when I go to the reports, the value of the returned product (1 from 2) is not shown: http://d.pr/i/1diTe. 

I tried to set the order as refunded/returned but when I do that, it automatically puts both of the products as refunded, not just one of them as I want. ;)

So this is our issue. 

Any ideas? 

Cheers'
